### PR TITLE
Clarify what happens after authentication.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue #61 by using @selfissued's language to clarify what happens after authentication.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/controller-document/pull/122.html" title="Last updated on Nov 16, 2024, 9:00 PM UTC (d6b4a7a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/controller-document/122/f2baa07...d6b4a7a.html" title="Last updated on Nov 16, 2024, 9:00 PM UTC (d6b4a7a)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1610,10 +1610,10 @@ <h2>Verification Relationships</h2>
...
 
             <p>
-The `authentication` [=verification relationship=] is used to
-specify how the [=controller=] is expected to be [=authenticated=], for
-purposes such as logging into a website or engaging in any sort of
-challenge-response protocol.
+The `authentication` [=verification relationship=] is used to specify how the
+[=controller=] is expected to be [=authenticated=], for purposes such as logging
+into a website or engaging in any sort of challenge-response protocol. The
+processing performed following authentication is application-specific.
```
Comment (dlongley) [inline index.html:1616]:
```suggestion
The `authentication` [=verification relationship=] is used to specify how the
[=subject=] is expected to be [=authenticated=], for purposes such as logging
into a website or engaging in any sort of challenge-response protocol. The
processing performed following authentication is application-specific.
```

Comment (dlongley) [reply index.html:1616]: The text below should also be fixed:

```
Note that the [=verification method=] indicated by the
`authentication` property of a [=controller document=] can
only be used to [=authenticate=] the [=controller=]. To
[=authenticate=] a different [=controller=], the entity associated with
the value of `controller` needs to [=authenticate=] with its
<em>own</em> [=controller document=] and associated
`authentication` [=verification relationship=].
```

Should change to:

```
Note that the [=verification method=] indicated by the
`authentication` property of a [=controller document=] can
only be used to [=authenticate=] the [=subject=]. To
[=authenticate=] a [=controller=], the entity associated with
the value of `controller` needs to [=authenticate=] with its
<em>own</em> [=controller document=] and associated
`authentication` [=verification relationship=].
```

**Review by David-Chadwick — COMMENTED** [review]
it is really important to be clear whether the authentication relationship refers to the controller or to the subject, given that they are not always the same entity. I think @dlongley is correct and it should be the subject

**Comment by dlongley** [comment]
@David-Chadwick,

Yes -- and I had another comment on here that I removed because Joe's PR #116 will also help clarify: https://github.com/w3c/controller-document/pull/116#discussion_r1845223102 ... and I don't want to create more churn here.

**Review by brianorwhatever — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2024-11-20-vcwg#section1-4) on 2024-11-20

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.4. Clarify what happens after authentication. (pr controller-document#122)

_See github pull request [controller-document#122](https://github.com/w3c/controller-document/pull/122)._




**Brent Zundel:** A couple more comments and a few fewer approvals than last one.  

**Manu Sporny:** Another from Mike Jones, he felt like the description wasn't adequate. I have tried to apply some of his language to this to clarify the definition.  

**Brent Zundel:** Please review and if you have comments, we can discuss, otherwise onto the next one.  

</details>

**Review by brentzundel — APPROVED** [review]

**Review by selfissued — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**